### PR TITLE
adding longshot version 0.4.5

### PR DESCRIPTION
**Pull request by erinyoung**
To make a "long" story short, this is dependency for another image that I'd like to create.

Longshot is a variant caller that specializes in noisy reads - like those of pacbio or nanopore.

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by erinyoung** [comment]
@Kincekara , do you have time to review this?

**Comment by erinyoung** [comment]
You can check the status of the deploy at https://github.com/StaPH-B/docker-builds/actions/runs/6710569877

Thank you for your contribution!!!

**Comment by Kincekara** [comment]
@erinyoung  Sorry, I didn't see your message. I am checking it now.

File: longshot/0.4.5/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# longshot container
+
+Main tool : [longshot](https://github.com/pjedge/longshot)
+
+Code repository: https://github.com/pjedge/longshot
```
Comment (Kincekara) [inline longshot/0.4.5/README.md:5]: I modified Read Me to comply with new templates and trigger checks.

**Review by Kincekara — APPROVED** [review]

**Comment by Kincekara** [comment]
I merged and deployed your PR. 
You can follow the progress from https://github.com/StaPH-B/docker-builds/actions/runs/6732773244.
Thank you for your contribution!

**Comment by kapsakcj** [comment]
TY @Kincekara !

**Comment by erinyoung** [comment]
Thank you!

**Pull request merged**